src/test/java/com/infobip/jira/CommitMessageVersionExtractorTest.java

```diff
@@ -48,7 +49,7 @@ public void shouldSuccessfullyExtractVersionFromReleaseMessage() {
 
         whenExtractVersion("Release 1.0.0");
 
-        then(version).isEqualTo(Optional.of("1.0.0"));
+        then(version).contains("1.0.0");
```

Comment:
OptionalAssert#contains verifies that they are equal (version is Optional here).
I've removed the Optional field though as Optionals shouldn't be used on fields. It was Optional before because guava Optional was used.